Libraries/Opc.Ua.Gds.Server.Common/ApplicationsNodeManager.cs

```diff
@@ -606,6 +609,44 @@ out nextRecordId
             return ServiceResult.Good;
         }
 
+        private ServiceResult OnCheckRevocationStatus(
+        ISystemContext context,
+        MethodState method,
+        NodeId objectId,
+        byte[] certificate,
+        ref StatusCode certificateStatus,
+        ref DateTime validityTime)
+        {
+            //Check if connected using a secure channel
```

Comment:
Move to 
AuthorizationHelper.HasAuthenticatedSecureChannel(ISystemContext context)

Reply:
yes will come in #2338

Reply:
merged with master, so the security check can be added

Reply:
done